packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -105,19 +110,26 @@
     }
   }
 
-  $: fetchRows(searchTerm, primaryDisplay)
+  $: fetchRows(searchTerm, primaryDisplay, defaultValue)
 
-  const fetchRows = (searchTerm, primaryDisplay) => {
+  const fetchRows = async (searchTerm, primaryDisplay, defaultVal) => {
     const allRowsFetched =
       $fetch.loaded &&
       !Object.keys($fetch.query?.string || {}).length &&
       !$fetch.hasNextPage
-    // Don't request until we have the primary display
-    if (!allRowsFetched && primaryDisplay) {
-      fetch.update({
-        query: { string: { [primaryDisplay]: searchTerm } },
+    // Don't request until we have the primary display or default value has been fetched
+    if (allRowsFetched || !primaryDisplay) {
+      return
+    }
+    if (defaultVal) {
+      await fetch.update({
+        query: { equal: { _id: defaultValue } },
       })
+      fetchedDefault = $fetch.rows?.[0]
```

Comment:
Why do we have this as a different object, instead of adding it to the `optionsObj`?

Reply:
Good call - didn't realise this was an option!

Reply:
Much nicer! Thx :D